inc/functions/cloudflare.php

```diff
@@ -51,10 +51,24 @@ function rocket_is_api_keys_valid_cloudflare( $cf_email, $cf_api_key, $cf_zone_i
 
 	try {
 		$cf_api_instance = new Cloudflare\Api( $cf_email, $cf_api_key );
-		$cf_user         = $cf_api_instance->get( 'user/' );
 		$cf_zone         = $cf_api_instance->get( 'zones/' . $cf_zone_id );
 
 		if ( ! isset( $cf_zone->success ) || empty( $cf_zone->success ) ) {
+			foreach ( $cf_zone->errors as $error ) {
+				if ( $error->code === 6003 ) {
```

Comment:
can this property be undefined?

Reply:
Based on CF docs, nope. An error object will contain an integer  `code` field and a `message`

https://api.cloudflare.com/#getting-started-responses